test/unit/annotation_spec.js

```diff
@@ -689,6 +689,34 @@ describe('annotation', function() {
       expect(data.dest).toEqual([{ num: 17, gen: 0, }, { name: 'XYZ', },
                                  0, 841.89, null]);
     });
+
+    it('should correctly parse a Dest, which violates the specification ' +
+       'by containing a dictionary', function() {
```

Comment:
Can you add a comment to the line where the invalid dictionary is constructed, for those who don't know the PDF spec by heart?

Reply:
Sorry, I don't completely follow what sort of comment you're asking for here. Are you simply asking for a reference to the particular section of the PDF spec, or something else?

> [...] where the invalid dictionary is constructed,

*I might be reading too much into this, but just to clarify:*
Please note that there's no way for a `Dest` entry to contain a "valid" dictionary, the mere fact that one is even present makes it invalid according to the spec.

Reply:
I was thinking of a comment before line 703, something like this (edit as needed):

```javascript
// Dest must be a Name or Array (PDF27000 section x.y.z), but there are PDFs that store a dictionary.
```